examples/TransferList.main.tsx

```diff
@@ -56,7 +56,7 @@ export default () => {
   return (
     <TransferList style={{ width: 500 }}>
       <TransferList.ListboxWrapper>
-        <TransferList.Listbox>
+        <TransferList.Listbox aria-label='listbox'>
```

Comment:
don't use `aria-label`. instead, add `TransferList.Label`.

Reply:
can you also change the label names to be more relevant?

this example is very contrived, but it would still help if it said "Column A"/"Column B" instead of just "Listbox".